addon/-private/modifier-classic.js

```diff
@@ -12,8 +13,11 @@ const Modifier = EmberObject.extend({
 
 Modifier.reopenClass({
   modifier(Klass) {
-    return Ember._setModifierManager(createManager, Klass);
+    deprecate("Modifier.modifier is deprecated.  Export the class directly.  See https://github.com/sukima/ember-oo-modifiers/pull/8", false, { id: 'modifier-call', until: "1.0.0" });
+    return Klass;
   }
 });
 
+Ember._setModifierManager(createManager, Modifier);
```

Comment:
Has this been replaced with a non-underscored method (via an import)? If so, should we make that change in this PR?

Reply:
Let's separate it out so we don't block this PR.